src/xr/xr-input-source.js

```diff
@@ -224,6 +224,9 @@ Object.assign(pc, function () {
     XrInputSource.prototype._updateTransforms = function () {
         var dirty;
 
+        if (! this._manager.active)
+            return;
+
         if (this._dirtyLocal) {
             dirty = true;
```

Comment:
And this occurrence?

Reply:
It happens when session is ended, but due to WebXR API async nature of session end event, it is possible to call `app.xr.end()` but then getting actual callback of XRSession end later. Although camera pointer is removed immediately on `end` call. So input sources are not removed immediately as they are removed by XRSession events.

We can start removing them by hand when `end` is manually executed.